FrontEnd/src/pages/AdminPage/UserProfilesTable/UserTable.jsx

```diff
@@ -66,7 +69,68 @@ function UserTable() {
             <CaretDownOutlined className={css['icon']} />
         );
     };
-
+    const handleSearch = (selectedKeys, confirm, dataIndex) => {
+        confirm();
+        setSearchText(selectedKeys[0]);
+        setSearchedColumn(dataIndex);
+  };
+    const handleReset = (clearFilters) => {
+        clearFilters();
+        setSearchText('');
+    };
+    const getColumnSearchProps = (dataIndex) => ({
+        filterDropdown: ({setSelectedKeys, selectedKeys, confirm, clearFilters}) => (
+            <div style={{padding: 8}}>
```

Comment:
Could we use scss instead of inline styles?

Reply:
Fixed